### PR TITLE
WIP: ttl migration tool

### PR DESCRIPTION
**Pull request by woodsaj**
do not merge.

### CODE REVIEW

File: cmd/mt-retention-migration/main.go [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+	sourceTTLTable := mdata.GetTTLTables(
+		[]uint32{sourceTTL},
+		20,
+		mdata.Table_name_format,
+	)
+	fmt.Println(
+		fmt.Sprintf("Using %s as source table", sourceTTLTable[sourceTTL].Table))
+
+	// output retentions
+	ttls := make([]uint32, 3)
```
Comment (woodsaj) [inline cmd/mt-retention-migration/main.go]: should this be created with initial size of 2?

Comment (replay) [inline cmd/mt-retention-migration/main.go]: right, forgot to update that after the output retention requirement change

**Comment by Dieterbe** [comment]
@woodsaj do we need this for anything? can we delete the PR and the branch ?

**Comment by woodsaj** [comment]
This was a one-off tool to fix up a customer.  So i dont think so.

**Pull request closed without merging**